crates/factor-outbound-pg/src/lib.rs

```diff
@@ -6,6 +6,7 @@ use spin_factors::{
     SelfInstanceBuilder,
 };
 use tokio_postgres::Client;
+use std::marker::Send;
```

Comment:
This is part of the automatic rust "prelude".

```suggestion
```

Reply:
Ahh! Will have to change for the `factor_outbound_redis` one too 😅 . might as well add to this PR!